book/src/SUMMARY.md

```diff
@@ -36,6 +36,7 @@
   - [Data Plane Fanout](data-plane-fanout.md)
   - [Reliable Vote Transmission](reliable-vote-transmission.md)
   - [Bank Forks](bank-forks.md)
+  - [Fullnode Changes](fullnode-with-ledger-notifications.md)
```

Comment:
This needs a better name. Every proposal we have fits in the general category of "Fullnode Changes". How about Blocktree Validation?

Reply:
👍